elks/arch/i86/kernel/irq.c

```diff
@@ -32,24 +34,7 @@ struct irqaction {
 	void *dev_id;
```

Comment:
Delete dev_id, always unused and delete struct irqaction, as what we use now is just array of handler pointers.

Reply:
Done in 990a917